PyRDF/Proxy.py

```diff
@@ -2,6 +2,9 @@
 from .backend import Executor
 from .CallableGenerator import CallableGenerator
 
+Backend_env = "local"
+Backend_conf = {}
```

Comment:
Also I would follow the convention of variables names starting with lowercase.

Reply:
:+1: